lib/builder-webpack4/src/preview/iframe-webpack.config.ts

```diff
@@ -238,7 +238,7 @@ export default async (options: Options & Record<string, any>): Promise<Configura
       modules: ['node_modules'].concat(envs.NODE_PATH || []),
       mainFields: [modern ? 'sbmodern' : null, 'browser', 'module', 'main'].filter(Boolean),
       alias: {
-        ...themingPaths,
+        ...(features.emotionAlias ? themingPaths : {}),
```

Comment:
Do we need `features?.x`? I think we encountered this in the Vite builder before

Reply:
Well, it shouldn't be nessecary because:
https://github.com/storybookjs/storybook/blob/e10f5d36f156d14b43e3ceff2997e0f1db2f60e6/lib/core-server/src/presets/common-preset.ts#L64-L68